src/app/components/form-mesas/form-mesas.component.ts

```diff
@@ -69,15 +75,20 @@ export class FormMesasComponent{
     }
   }
 
-  modificarMesa()
+  async modificarMesa()
   {
     console.log("Modificar Mesa");
 
     if(this.mesa)
```

Comment:
no ves mejor un ->
if(this.mesa) && if(this.auxiliarFoto)
para evitar if anidados

Reply:
No, porque uno puede modificar o no la imagen de una mesa, es opcional.